app/assets/locales/android_translatable_strings.txt

```diff
@@ -381,6 +381,15 @@ notification.sync.connections.title=No Connection
 notification.sync.connections.detail=No active data connections
 notification.sync.connections.action=You don't have any active data connections configured. Try connecting your phone to wifi or turning mobile data on.
 
+
+connection.captive_portal.title=No Internet Detected
+connection.captive_portal.detail=Cannot access internet
+connection.captive_portal.action=You are in a wifi that requires sign-in from a browser. Try authenticating using browser or turning mobile data on.

```

Comment:
Nitpick: "You are connected to a network that requires sign-in from a browser. Try authenticating using browser or switching to mobile data"